File: JavaFundamentals/DataTypes/Exercise/SumDigits.java
package FundamentalsModule.DataTypes.Exercise;

import java.util.Scanner;
public class SumDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int input = Integer.parseInt(scanner.nextLine());
        int sum=0;

        for(int i =0; i<String.valueOf(input).length(); i++){
            sum=sum+Integer.parseInt(String.valueOf(String.valueOf(input).charAt(i)));
        }
        System.out.println(sum);


    }
}

File: JavaScriptBasics/ForCycle/Lab/SumDigits.js
function SumDigits(input){

let sum = 0;
let number = input[0];

for(let i=0; i<number.length; i++){

    let digit = Number(number[i]);
    sum = sum+digit;
  
}
console.log("The sum of the digits is:"+sum);

}